consensus/src/chained_bft/test_utils/mock_txn_manager.rs

```diff
@@ -13,35 +13,29 @@ use std::sync::{
 pub type MockTransaction = usize;
 
 /// Trivial mock: generates MockTransactions on the fly. Each next transaction is the next value.
+#[derive(Clone)]
 pub struct MockTransactionManager {
-    next_val: AtomicUsize,
+    next_val: Arc<AtomicUsize>,
```

Comment:
This is probably worth a comment explaining that we need this so that MockTransactionManager is Clone.